wf-hashsplitter/Snakefile

```diff
@@ -5,16 +5,16 @@ localrules: all, download
 # any other remote storage is possible as well e.g. FTP
 # Remote storage has to be suported by Snakemake.
 # https://snakemake.readthedocs.io/en/stable/snakefiles/remote_files.html
-S3 = S3RemoteProvider(access_key_id="XXX", secret_access_key="XXX", endpoint_url="https://s3-elixir.cloud.e-infra.cz")
+S3 = S3RemoteProvider(access_key_id="8vVw4pzA4CLAg4VI", secret_access_key="yI2j5Y4IDEqWEGyewnmGUv4dXiGZy7OR", endpoint_url="https://s3-elixir.cloud.e-infra.cz")
```

Comment:
why did you add the credentials?

Reply:
Oopsie. I'll fix it in a minute 🙈 One other reason why I don't like this approach by Snakemake, this is so easily forgotten